src/systemTest/resources/README-JDK7.txt

```diff
@@ -0,0 +1,8 @@
+To generate the methods files 'JDK7-sef-methods.txt' and 'JDK7-omitmethods.txt':
+
+- JDK7-sef-methods.txt: Copy the generated JDK-sef-methods.txt from the output of the 
+    MethodListGen tool. Manually remove the files present in JDK8 but not JDK7.
```

Comment:
Which files are these?  Can you provide a grep command that does this?

Reply:
I'll clarify this.